tools/run_local_http_server.py

```diff
@@ -0,0 +1,33 @@
+#  Copyright 2023 Google LLC.
+#  Copyright (c) Microsoft Corporation.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import sys
+from pathlib import Path
+
+from pytest_httpserver import HTTPServer
+
+# Current directory is not a module, so to import `LocalHttpServer`, its path
+# has to be added to `sys.path`. It is done relative to this file's directory.
+# The `flake8` is disabled for this reason.
+sys.path.append(str(Path(__file__).resolve().parent / '../tests/tools/'))
+
+import local_http_server  # noqa: E402
+
+httpserver = HTTPServer()
+httpserver.start()
+local_http_server = local_http_server.LocalHttpServer(httpserver)
+print("Local http server started...")
```

Comment:
Just a slightly improved suggestion:

```
print(f"""
Local http server started...
  - 200: {...}
  - 301 / permanent redirect: {...}
"""

And maybe also print in the first line where the server started? E.g. localhost:8080 or similar.
```

Reply:
Write where the server runs requires adding it in the API, and I don't think it is needed for tests. The developer can easily see the address part in the lines below.